backend/game.py

```diff
@@ -42,79 +42,89 @@ def __init__(self, game_id: Text, options: GameOptions):
         self._game_id = game_id
 
     def play(self, tribe1: Tribe, tribe2: Tribe, gamedb: Database, engine: Engine) -> Player:
+        game_id = ""
+        if hasattr(self, '_game_id'):
+            game_id = self._game_id
+
         last_tribe_standing = self._play_multi_tribe(tribe1=tribe1, tribe2=tribe2,
                                                      gamedb=gamedb, engine=engine)
-        log_message("Last tribe standing is {}.".format(last_tribe_standing))
+        log_message("Last tribe standing is {}.".format(last_tribe_standing), game_id)
```

Comment:
use named arguments for readability i.e.

log_message(message="...", game_id="...")